app/internal_packages/message-list/lib/message-participants.tsx

```diff
@@ -34,18 +33,25 @@ export default class MessageParticipants extends React.Component<MessageParticip
   }
 
   _shortNames(contacts = [], max = MAX_COLLAPSED) {
-    let names = contacts.map(c =>
-      c.displayName({
-        includeAccountLabel: true,
-        compact: !AppEnv.config.get('core.reading.detailedNames'),
-      })
-    );
+    let names = contacts.map((c, i) => (
+      <span key={`contact-${i}`}>
+        {i > 0 && ', '}
+        <span onContextMenu={() => this._onContactContextMenu(c)}>
+          {c.displayName({
+            includeAccountLabel: true,
+            compact: !AppEnv.config.get('core.reading.detailedNames'),
+          })}
+        </span>
+      </span>
+    ));
+
     if (names.length > max) {
       const extra = names.length - max;
       names = names.slice(0, max);
-      names.push(`and ${extra} more`);
+      names.push(<span>and ${extra} more</span>);
```

Comment:
This is a super minor nit, but I think this line may need a `key="contact-more"` prop to silence a React warning because it's rendered out of an array.